department-of-veterans-affairs/caseflow
# Add Edit NOD Modal and Timeline Link

Resolves #15499 

### Description
Add timeline link and 'Edit NOD' modal component. Covers 196.1-196.3

### Acceptance Criteria
- [ ] Link for 'Edit NOD' in the timeline section on the Case Details page
- [ ] Link 'Edit NOD' opens edit NOD modal
- [ ] Default date for Date Selector component is set to original NOD date (196.3)
- [ ] Success alert appears after submitting

### Testing Plan
1. Search for a Veteran's SSN
2. Click "Docket Number" and scroll to Case Timeline Section on the Case Details page
3. Verify modal appears after clicking "Edit NOD"
4. Select a date and click "Submit"
5. Verify success alert appears in Case Timeline section

### User Facing Changes
 - [ ] Edit NOD Link in Timeline component

 BEFORE|AFTER
 ---|---
<img width="763" alt="Screen Shot 2020-11-10 at 11 39 40 PM" src="https://user-images.githubusercontent.com/68879427/98783193-04fe7a00-23ae-11eb-9f32-da05ff28ae68.png">| <img width="900" alt="Screen Shot 2020-11-10 at 11 27 06 PM" src="https://user-images.githubusercontent.com/68879427/98782686-493d4a80-23ad-11eb-9eb4-5f9ca6edcb12.png"><img width="914" alt="Screen Shot 2020-11-10 at 11 32 53 PM" src="https://user-images.githubusercontent.com/68879427/98782698-50645880-23ad-11eb-8b00-44e2d5ae7b9e.png">


### Storybook Story
[ ] Story exists for EditNODModal.jsx in EditNODModal.stories.js
